raiden/tests/scenarios/bf4_non_allowed_ops.yaml

```diff
@@ -0,0 +1,151 @@
+    default-reveal-timeout: 20
+
+# This is the bf4 scenario. It aims to check that operations that should not be possible to perform, are indeed not possible
+# to perform. It does so by trying some different unallowed operations such as opening the same channel twice, withdrawing more
+# than a node is allowed to, depositing more than the on chain balance, making payments after a channel is closed etc.
+
+scenario:
+  serial:
+    tasks:
+      - parallel:
+          name: "Open channel"
+          tasks:
+            - open_channel: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, expected_http_status: 201}
+      - parallel:
+          name: "Assert after channel opening"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+      - parallel:
+          name: "Deposit in the other direction"
+          tasks:
+            - deposit: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, expected_http_status: 200}
+      - parallel:
+          name: "Assert after deposit"
+          tasks:
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+      - serial:
+          name: "Try to open already opened channel, expected to not be allowed with 409"
+          tasks:
+            - open_channel: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, expected_http_status: 409}
+      - serial:
+          name: "Send payment of higher amount than available, expected 409"
+          tasks:
+            - transfer: {from: 0, to: 1, amount: 1_000_000_000_000_000_001, lock_timeout: 30, expected_http_status: 409}
+      - parallel:
+          name: "Assert that balance didn't change"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+      - serial:
+          name: "Node1 tries to withdraw more than its balance"
+          tasks:
+            - withdraw: {from: 1, to: 0, total_withdraw: 1_100_000_000_000_000_000, expected_http_status: 409}
+      - serial:
+          name: "Assert that balance didn't change"
+          tasks:
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+      - serial:
+          name: "Node1 withdraws it entire balance"
+          tasks:
+            - withdraw: {from: 1, to: 0, total_withdraw: 1_000_000_000_000_000_000, expected_http_status: 200}
+      - serial:
+          name: "Assert that balance changed"
+          tasks:
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 0, state: "opened"}
+      - serial:
+          name: "Node1 tries to withdraw again"
```

Comment:
This and the previous are tested here:

https://github.com/raiden-network/raiden/blob/a220f4a59604d55c6095186b0d739f3cf1e93777/raiden/tests/integration/api/test_restapi.py#L2358-L2382

That test can have a bit more obvious variable names to make it obvious that the second withdraw should fail. I will change that.